### PR TITLE
read api bytes logging, upgrade bigquery client versions

### PR DESCRIPTION
**Pull request by kcibul**
The main purpose of this PR was to output the new estimated bytes read from the Read API for better monitoring and debugging.

However in the course of that I discovered that we were using ancient versions of the google APIs.  No massive improvements from the release logs, but a lot of nice features (cleaner logging, automatic retries for certain errors, )

bigQuery 1.131.1 -> 2.3.3 [Release log for bigQuery](https://github.com/googleapis/java-bigquery/blob/main/CHANGELOG.md)

bigQueryStorage 1.22.3 -> 2.5.0 [Release log for bigQueryStorage](https://github.com/googleapis/java-bigquerystorage/blob/main/CHANGELOG.md)

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java [hunk 1]
```diff
@@ -341,7 +341,8 @@ protected void onShutdown() {
 
         if ( engine != null ) {
             logger.info(String.format("***Processed %d total sites", engine.getTotalNumberOfSites()));
-            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));
+            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));            logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:344]: this formatting looks kind of odd--maybe it's just github?

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateSiteFilteringFiles.java [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package org.broadinstitute.hellbender.tools.gvs.filtering;
 
 import htsjdk.variant.variantcontext.VariantContext;
-import io.grpc.netty.shaded.io.netty.util.internal.StringUtil;
 
+import io.netty.util.internal.StringUtil;
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateSiteFilteringFiles.java:5]: why is this change needed? or is it part of one of the upgrades?

**Review by RoriCremer — APPROVED** [review]

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateSiteFilteringFiles.java (hunk 2 above)
Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateSiteFilteringFiles.java:5]: We were importing the "shaded" version of this before which isn't really the library, but gradle's repackaging of it and it's not stable.  This is the public version.

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java (hunk 1 above)
Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:344]: oh yeah that is strange…

Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java]: fixed

**Review by rsasch — APPROVED** [review]
LGTM — are you planning to build a jar after this is merged so we can start to use it for beta, or is this a post-beta thing?

**Comment by kcibul** [comment]
I'd like it to be a beta thing… I'm just working through some gradlew stuff
now
-------------------------------
Kristian Cibulskis
Director, Technology - SpecOps
Broad Institute of MIT and Harvard
***@***.***
